widget.md

```diff
@@ -0,0 +1,48 @@
+## Quick Start
+
+This embedded `<iframe>` allows you to embed a Harvest Timer form directly into your application. Create an `<iframe>` like this:
+
+```
+<iframe 
+  src="https://platform.harvestapp.com/platform/timer?
+    app_name=ExampleCompany&
+    closable=false&
+    permalink=https%3A%2F%2Fexample.com%2Fitem%2F1&
+    external_item_id=1&
```

Comment:
[nb] Just a thought. What if we dropped the `external_` prefix? It's not really useful plus it makes the param names twice as long?